cmd/hatchet-migrate/migrate/migrations/20250903152758_v1_0_41.sql

```diff
@@ -0,0 +1,57 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE v1_dag_to_task_partitioned (
+    dag_id BIGINT NOT NULL,
+    dag_inserted_at TIMESTAMPTZ NOT NULL,
+    task_id BIGINT NOT NULL,
+    task_inserted_at TIMESTAMPTZ NOT NULL,
+    PRIMARY KEY (dag_id, dag_inserted_at, task_id, task_inserted_at)
+) PARTITION BY RANGE(dag_inserted_at);
+
+SELECT create_v1_range_partition('v1_dag_to_task_partitioned', NOW()::DATE);
+SELECT create_v1_range_partition('v1_dag_to_task_partitioned', (NOW() + INTERVAL '1 day')::DATE);
```

Comment:
Shouldn't there be corresponding code changes to maintain this partition?

Also: would weekly partitions make sense here, or daily?

Reply:
yup, thanks 🤦 keep forgetting this when going from cloud to the oss

re: weekly vs daily - I guess weekly is fine? it'd be easier on the planner